src/libs/PusherConnectionManager.ts

```diff
@@ -1,30 +1,29 @@
-import lodashGet from 'lodash/get';
+import {ValueOf} from 'type-fest';
 import * as Pusher from './Pusher/pusher';
 import * as Session from './actions/Session';
 import Log from './Log';
 import CONST from '../CONST';
 
+type EventCallbackError = {type: ValueOf<typeof CONST.ERROR>; data: {code: number}};
+
 function init() {
     /**
      * When authTokens expire they will automatically be refreshed.
      * The authorizer helps make sure that we are always passing the
      * current valid token to generate the signed auth response
      * needed to subscribe to Pusher channels.
      */
-    Pusher.registerCustomAuthorizer((channel) => ({
-        authorize: (socketID, callback) => {
+    Pusher.registerCustomAuthorizer((channel: {name: string}) => ({
```

Comment:
Could you extract `{name: string}` into a separate type?

Reply:
Done! :) @fabioh8010